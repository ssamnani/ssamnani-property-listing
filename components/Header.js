import React from "react";
import "./Header.module.css";
import HeaderTabs from "./HeaderTabs";
import Breadcrumb from "./Breadcrumb";

function Header() {
  return (
    <div>
      <div className="appHeader">
        <div className="navWrapper">
          <img
            src="/assets/images/iproperty-logo.png"
            className="logo"
            alt="iproperty-logo"
          />
          <div>
            <HeaderTabs />
          </div>
        </div>
        <Breadcrumb />
      </div>
    </div>
  );
}

export default Header;
