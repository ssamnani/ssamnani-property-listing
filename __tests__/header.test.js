import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Header from "../components/Header";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render("<Header />", div);
});

// Snapshot Testing
it("matches snapshot", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
