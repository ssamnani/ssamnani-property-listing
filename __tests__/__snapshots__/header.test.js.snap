// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    className="appHeader"
  >
    <div
      className="navWrapper"
    >
      <img
        alt="iproperty-logo"
        className="logo"
        src="/assets/images/iproperty-logo.png"
      />
      <div>
        <div
          className="headerTabs"
        >
          <ul>
            <li>
              <a
                className="active"
              >
                Buy
              </a>
            </li>
            <li>
              <a>
                Rent
              </a>
            </li>
            <li>
              <a>
                New Launches
              </a>
            </li>
            <li>
              <a>
                Advertise
              </a>
            </li>
            <li>
              <a>
                News & Lifestyle
              </a>
            </li>
            <li>
              <a>
                Events
              </a>
            </li>
            <li>
              <a>
                International
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="breadcrumbWrapper"
    >
      <ol>
        <li>
          Home
        </li>
        <li>
          Sale
        </li>
        <li>
          All Residential
        </li>
      </ol>
    </div>
  </div>
</div>
`;
